src/client.ts

```diff
@@ -42,7 +44,8 @@ export const createClient = (args: AuthProps, opts?: ConstructorOpts) => {
     // if a function to fetch email+token from AWS was provided, try that
     else if (args.getAwsParameterStoreName) {
       const parameterName = args.getAwsParameterStoreName(subdomain)
-      const ssm = new SSM()
+      // Use custom AWS region if provided
+      const ssm = args.awsRegion ? new SSM({ region: args.awsRegion }) : new SSM()
```

Comment:
Is it possible just to do `new SSM({ region: args.awsRegion })` even if it is undefined?

Reply:
I'm not sure, I went with this to be safe. Will try to see what happens if the region is undefined.